libcodechecker/cmd/cmd_line_client.py

```diff
@@ -239,14 +239,23 @@ def add_filter_conditions(report_filter, filter_str):
     severity, checker, path = map(lambda x: x.strip(), filter_str.split(':'))
 
     if severity:
-        report_filter.severity = \
-            shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
+        report_filter.severity\
+            = shared.ttypes.Severity._NAMES_TO_VALUES[severity.upper()]
     if checker:
         report_filter.checkerId = '*' + checker + '*'
     if path:
         report_filter.filepath = path
 
 
+def add_server_arguments(parser):
```

Comment:
I have an upcoming merge request that changes the `cmd` subcommand to the new structure. The change irrelevant to the implementation of `suppress` sub-subcommand makes the previously created diffs a lot harder to manage.

But as you see fit, a rebase would be essential anyways for me later on.